### ISSUE TITLE
Invalid syntax

### ISSUE THREAD

**Issue by nn-user** [aio-libs/aiohttp#3521]
## Long story short

```
Traceback (most recent call last):
  File "main.py", line 1, in <module>
    import aiohttp
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/__init__.py", line 10, in <module>
    from .protocol import *  # noqa
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/protocol.py", line 18, in <module>
    from .helpers import reify
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/helpers.py", line 25
    ensure_future = asyncio.async
                                ^
SyntaxError: invalid syntax
Error in sys.excepthook:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/apport_python_hook.py", line 63, in apport_excepthook
    from apport.fileutils import likely_packaged, get_recent_crashes
  File "/usr/lib/python3/dist-packages/apport/__init__.py", line 5, in <module>
    from apport.report import Report
  File "/usr/lib/python3/dist-packages/apport/report.py", line 30, in <module>
    import apport.fileutils
  File "/usr/lib/python3/dist-packages/apport/fileutils.py", line 23, in <module>
    from apport.packaging_impl import impl as packaging
  File "/usr/lib/python3/dist-packages/apport/packaging_impl.py", line 24, in <module>
    import apt
  File "/usr/lib/python3/dist-packages/apt/__init__.py", line 23, in <module>
    import apt_pkg
ModuleNotFoundError: No module named 'apt_pkg'

Original exception was:
Traceback (most recent call last):
  File "checker.py", line 1, in <module>
    import aiohttp
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/__init__.py", line 10, in <module>
    from .protocol import *  # noqa
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/protocol.py", line 18, in <module>
    from .helpers import reify
  File "/usr/local/lib/python3.7/dist-packages/aiohttp/helpers.py", line 25
    ensure_future = asyncio.async
```

## Expected behaviour

Not crash my script

## Actual behaviour

Crash, not worked with python3.7

## Steps to reproduce

Used module aiohttp

## Your environment

I use this module in my program, but after the update it stopped working

**Comment by Harmon758** [comment]
What version of aiohttp are you using?
This specific issue was fixed in v2.3.3 with https://github.com/aio-libs/aiohttp/commit/7e15124e78d3ec9e6752fbde51671480813d46ba more than a year ago.
Anything resembling this line was removed in v3.0.0 with https://github.com/aio-libs/aiohttp/commit/1a606bc1f08db2291a0d4dcd6a8b600e502ebc70 when Python 3.4 support was dropped.
Please make sure you're up to date with the latest development version when submitting issues.

**Comment by nn-user** [comment]
Understood the reason, sorry to bother you.
